test/test_ops.py

```diff
@@ -85,6 +86,9 @@
 
 assert torch.get_default_dtype() == torch.float32
 
+FakeTensorConfig.cache_enabled = True
+FakeTensorConfig.cache_crosscheck_enabled = True
```

Comment:
Maybe it makes more sense to just patch this within `TestFakeTensor` as well as `test_fake_tensor.py`